### ISSUE TITLE
Increase legibility and enable typing above 5wpm

### ISSUE THREAD

**Issue by jimbo1qaz** [FourMan/4#3]
In other words, stop abusing emojis. Emoji is for idiots.

**Comment by NPException42** [comment]
I suggest a closing this issue with "won't fix".

**Comment by howdoicomputer** [comment]
I highly suggest closing this issue as "won't fix".

**Comment by stemid** [comment]
My suggestion is closing this issue with "will not fix".

**Comment by jimbo1qaz** [comment]
💩

**Comment by Intrepidity** [comment]
+1
Aside of a few giggles this cannot serve a practical purpose whatsoever.

**Comment by yanislavb** [comment]
I like programming in easy level ;)
I like emoji and I hope 4chan will done this project on next month ;)
In other words , I am lazy ass xD

**Comment by jiahao** [comment]
You could adopt Julia's LaTeX-like tab completion syntax. For example, \:koala:<TAB> gives 🐨. JuliaLang/julia#10709

**Comment by FourMan** [comment]
@jimbo1qaz: teaching idiots to program would be great, actually
Legibility and typing speed is going to turn out great for 🍀 since we have planned a custom keyboard with a custom font. But since we are focusing on the actual language and compiler, closing this

@jiahao: thanks, taking a look. Have stumbled on more emoji-related programming projects than expected before we started this project.

**Comment by jimbo1qaz** [comment]
Non-programmers are not idiots. Teenage girls seemingly incapable of typing anything other than emoji in text messages are idiots.

**Comment by FourMan** [comment]
@jimbo1qaz:
I think it would be worthwhile to teach them a new skill since they are already used to typing as such.
